src/Writer/Entity/Product.php

```diff
@@ -100,7 +100,7 @@ public function setTax(?float $tax): void
             return;
         }
 
-        if ($tax == 0) {
+        if (empty($tax)) {
```

Comment:
`strict_comparison` wanted this to be `===` but since `$tax` is `float`, this also covers `$tax` being `0.00` for example.